bbn-test-4/finality-providers/README.md

```diff
@@ -226,10 +231,38 @@ being the same as the finality provider information stored under `./finality-pro
 but with the `.sig` extension (e.g. `${nickname}.sig`).
 The content of the file should be the plain value of the `schnorr_signature_hex` field.
 
+The signature was generated by reading the entire file data, not only the file
+content. For proper verification, the exactly file used for signing should
+be submited in the pull request.
+
 ## 5. Create Pull Request
 
 Submit your finality provider information under the `registry` directory and
 your signature under the `sigs` directory. Both file names should have the
 same name (e.g. `${nickname}`), but with `.json` and `.sig` extensions respectively.
-Make sure that you submit exactly the same file that you signed to ensure proper
-verification.
+**Make sure that you submit exactly the same file that you signed to ensure proper
+verification**.
+
+If you have installed all the binaries in your path, check out locally if your
+finality provider is valid before creating the pull request:
```

Comment:
```suggestion
If you have installed all the binaries in your path, you can check out locally if your
finality provider is valid before creating the pull request,
by using the following script contained in this repo
```

Reply:
We should also have instructions for verifying that the deposit transaction is valid before sending it to BTC.

Reply:
I can split the verification into multiple files

Reply:
https://github.com/babylonchain/networks/pull/193/commits/0195703c03ffc6325ed9f286db69b230871197e7